web/models/invitation.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Entice.Web.Invitation do
+  use Ecto.Model
+
+  schema "invitations" do
+    field :email, :string
+    field :key, :string
+    field :createdat, Ecto.DateTime, default: Ecto.DateTime.local
```

Comment:
Is this used somewhere? Do we insert the current time or is this not defined when creating and invitation?